src/main/resources/agreement-and-default-flow/valid/audiences/manifest-sha1.txt

```diff
@@ -1,4 +0,0 @@
-0a66ea77834e337e28a043db6d6f3d745c944593  data/random images/image03.jpeg
```

Comment:
why do you remove the whole manifest and tagmanifest files for this example?

Reply:
recalculated the manifest.
But they were not tracked so i forgot to commit them.